modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,48 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::ampcombi=0.1.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.1.5--pyhdfd78af_0':
+        'quay.io/biocontainers/ampcombi:0.1.5--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(amp_input)
+    path(faa_folder)
+
+    output:
+    tuple val(meta), path("*/*")                       , emit: sample_dir
+    tuple val(meta), path("*/*diamond_matches.txt")    , emit: txt
+    tuple val(meta), path("*/*ampcombi.csv")           , emit: csv
+    tuple val(meta), path("*/*amp.faa")                , emit: faa
+    path("AMPcombi_summary.csv")                       , optional:true, emit: summary_csv
+    path("AMPcombi_summary.html")                      , optional:true, emit: summary_html
+    path("*.log")                                      , emit: log
+    path("*/amp_ref_database")                         , optional:true, emit: results_db
+    path("*/amp_ref_database/*.dmnd")                  , optional:true, emit: results_db_dmnd
+    path("*/amp_ref_database/*.clean.fasta")           , optional:true, emit: results_db_fasta
+    path("*/amp_ref_database/*.tsv")                   , optional:true, emit: results_db_tsv
```

Comment:
Why don't these have meta  if the earlier ones do?

Reply:
We did not put meta, because these files are not related specifically to one sample: The reference database may be used for several samples. The AMPcombi_summary files can only be generated, if AMPcombi parses a folder (like in the first test-workflow). I don't think the module will be applied with this option, but as the tool is able to do so, we included it.

Reply:
If it takes in meta you might as well pass out meta, you don't wnat to to assume what a user is doing.

You could have different batches of AMP outputs that you want to store separately.

Reply:
Alright, we added them to all outputs.